R/setup_conda.R

```diff
@@ -0,0 +1,29 @@
+#' Setup conda
+#' 
+#' Install miniconda and setup a conda env to use python dependencies.
+#' @inheritParams reticulate::install_miniconda
+#' @inheritParams reticulate::py_install
+#' @inheritDotParams reticulate::py_install
+#' @returns Null
+#' 
+#' @export
```

Comment:
I wouldn't export this function for now, at least until there is a reticulate backend. Also, we shouldn't create an Rd file for functions that aren't exported.

```suggestion
#' @noRd
```